qiskit-superstaq/qiskit_superstaq/compiler_output.py

```diff
@@ -218,7 +218,10 @@ def read_json(json_dict: Dict[str, str], circuits_is_list: bool) -> CompilerOutp
         pulse_gate_circuits = qss.deserialize_circuits(json_dict["pulse_gate_circuits"])
 
     if "pulses" in json_dict:
-        pulse_sequences = gss.serialization.deserialize(json_dict["pulses"])
+        try:
+            pulse_sequences = gss.serialization.deserialize(json_dict["pulses"])
+        except Exception:
+            pulse_sequences = None
```

Comment:
Just wondering -- would it be useful to have a warning raised (like in `css`) for the `Exception`?

Reply:
good question, i was debating this. My feeling was that it might be more confusing than valuable, in that we expect the user to use the pulse gate circuits (which should still be available when this fails) instead of the pulse schedules anyway - the schedules are mostly still attached to facilitate the diagrams in some of our demo notebooks, which will still work as long as they're run with the most recent qiskit version (and which we should be migrating to the timeline drawer after https://github.com/Qiskit/qiskit/issues/10582 is fixed)

Reply:
I see, that makes sense to me. PR looks good then!